src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.search-big {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.search-big__input {
+  width: 100%;
+  height: 70px;
+  padding-left: 63px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+.search-big__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-big__input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search-big input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
```

Comment:
do not style tags

move common font styles to `html` selector and use `inherit` then